src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -19,11 +22,15 @@ public class PythonLanguageService : LanguageProcessor
         private readonly string _pythonExecutablePath;
         public override string ProcessName => _pythonExecutablePath;
         private readonly string _apiScriptPath;
+        private readonly string _tempDiirectory;

```

Comment:
nit: typo _tempDirectory

Reply:
reverted this change to remove temp path